examples/Atmos/heldsuarez.jl

```diff
@@ -159,7 +159,7 @@ Base.@kwdef struct HeldSuarezSetup{FT}
   domain_height::FT = 30e3
 end
 
-function (setup::HeldSuarezSetup)(state, aux, coords, t) 
+function (setup::HeldSuarezSetup)(state, aux, coords, t, args...)
```

Comment:
Should only need to add these extra args if they're actually used.

Reply:
Ok, I’ll see if I can remove this without breaking